src/services/productServices/viewSingleProduct.ts

```diff
@@ -18,10 +18,6 @@ export const viewSingleProduct = async (req: Request, res: Response) => {
       if (!product) {
         return res.status(404).send({ status: 'error', message: 'Product not found' });
       }
-
-      if (product.expirationDate && new Date(product.expirationDate) < new Date()) {
-        return res.status(400).json({ status: 'error', message: 'Product expired' });
-      }
       res.status(200).json({ status: 'success', product: product });
```

Comment:
Why aren't we checking this anymore ? won't it cause having expired products in stock?

Reply:
this was preventing me from fetcting the products , because even if the product is expired a vendors still needs to see it . in the get all the items for a vendor these products are their so geting all the info should also be their , maybe we could put this to the generall search because this route is only for the vendor , and when checking the stock i think there is another check for that